fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -257,6 +257,171 @@ sebasedBiasFieldMNI="$SubjectFolder/$AtlasSpaceFolder/Results/$NameOffMRI/${Name
+                                                                         #   Errors in temporal interpolation will occur in the presence of head motion and may also disrupt 
+                                                                         #   data quality measures as shown in Power et al 2017 PLOS One "Temporal interpolation alters motion in fMRI
+                                                                         #   scans: Magnitudes and consequences for artifact detection." Slice timing correction and motion correction 
+                                                                         #   would ideally be performed simultaneously; however, this is not currently supported by any major software 
+                                                                         #   tool. HCP-Style fast TR fMRI data acquisitions (TR<=1s) avoid the need for slice timing correction, 
+                                                                         #   provide major advantages for fMRI denoising, and are recommended. 
+                                                                         #   No slice timing correction is done by default.  
+
+# If --dcmethod=NONE                                                     # WARNING: The fMRIVolume pipeline is being run without appropriate distortion correction of the fMRI image. 
+                                                                         #   This is NOT RECOMMENDED under normal circumstances. We will attempt 6 DOF FreeSurfer BBR registration of 
+                                                                         #   the distorted fMRI to the undistorted T1w image. Distorted portions of the fMRI data will not align with 
+                                                                         #   the cortical ribbon. In HCP data 30% of the cortical surface will be misaligned by at least half cortical 
+                                                                         #   thickness and 10% of the cortical surface will be completely misaligned by a full cortical thickness. 
+                                                                         #   At a future time, we may be able to add support for fieldmap-less distortion correction. At this time, 
+                                                                         #   however, despite ongoing efforts, this problem is unsolved and no extant approach has been successfully 
+                                                                         #   shown to demonstrate clear improvement according to the accuracy standards of HCP-Style data analysis when 
+                                                                         #   compared to gold-standard fieldmap-based correction.
+
+# Defaults
+
+fMRIReference=`opts_DefaultOpt $fMRIReference "NONE"`
+BOLDMask=`opts_DefaultOpt $BOLDMask "T1_fMRI_FOV"`
```

Comment:
Move up to be with the other "Defaults".

Reply:
I moved them to line 301. I hope that is what you meant.